Source/JavaScriptCore/runtime/RegExp.cpp

```diff
@@ -291,8 +292,9 @@ bool RegExp::matchConcurrently(
     return true;
 }
 
-void RegExp::compileMatchOnly(VM* vm, Yarr::CharSize charSize)
+void RegExp::compileMatchOnly(VM* vm, Yarr::CharSize charSize, std::optional<StringView> sampleString)
 {
+    UNUSED_PARAM(sampleString);
```

Comment:
Let's put this in the `#else` section below together with `UNUSED_PARAM(charSize)`.

Reply:
Changed